packages/ui/dialogs/my-account/index.tsx

```diff
@@ -362,24 +365,6 @@ class MyAccount extends Component<Props, State> {
         {authenticated ? <Button round onPress={() => this.setState({ step: 2 })} text={changePin} /> :
         <Button round onPress={() => this.setState({ step: 4 })} text={enterCurrentPin} />}
       </View>),
-      (<View style={style.spaceHorizontalL}>
-        <Text style={[style.text, style.spaceTopL, style.center]}>{setNickname}</Text>
```

Comment:
This text should only be displayed if the user does not have a nickname set, otherwise just don't show any text here at all.